biosys/apps/main/utils_data_package.py

```diff
@@ -745,7 +745,11 @@ def row_it(self):
         for record in self.records:
             row = []
             for field in self.schema.field_names:
-                row.append(six.u(record.data.get(field, '')))
+                value = record.data.get(field, '')
+                # TODO: remove that when running in Python3
+                if isinstance(value, six.string_types) and not isinstance(value, six.text_type):
+                    value = six.u(value)
+                row.append(value)
```

Comment:
The fix. In python 2.x you cannot unicode an already unicoded string. Throws an exception!